### PR TITLE
Unify all zkSync forks into a single monitor

### PR DESCRIPTION
**Pull request by sz-piotr**
Based on work by @sin6pi7 so I'm most excited about your review.

### CODE REVIEW

File: packages/update-monitor/package.json [hunk 1]
```diff
@@ -1,29 +1,21 @@
 {
-  "name": "@l2beat/zkswap-monitor",
+  "name": "@l2beat/update-monitor",
   "version": "0.1.0",
   "private": true,
   "scripts": {
+    "build": "yarn typechain",
```
[the remark this answers is not in the document]
Comment (sz-piotr) [reply packages/update-monitor/package.json:6]: Just realized this

File: packages/update-monitor/src/run.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import { writeFile } from 'fs/promises'
+
+import { getZkSpaceParameters } from './zkSpace'
+import { getZkSwap1Parameters } from './zkSwap1'
+import { getZkSwap2Parameters } from './zkSwap2'
+import { getZkSyncParameters } from './zkSync'
+
+export async function run() {
+  dotenv.config()
+  const alchemyApiKey = process.env.ALCHEMY_API_KEY
```
Comment (sin6pi7) [inline packages/update-monitor/src/run.ts:12]: Maybe it's worth adding a one-liner about the env var being necessary?

**Comment by sin6pi7** [comment]
What was the rationale for going with just typechain over eth-sdk?

**Comment by sz-piotr** [comment]
> What was the rationale for going with just typechain over eth-sdk?

I didn't have the need to use any of eth-sdk functions

**Review by sin6pi7 — APPROVED** [review]
Awesome 🙌

**Pull request merged**